src/System.Memory/src/Configurations.props

```diff
@@ -2,9 +2,10 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netstandard1.0-Windows_NT;
-      netstandard-Windows_NT;
+      netstandard1.0;
+      netstandard;
       netcoreapp-Windows_NT;
+      netcoreapp-Unix;
```

Comment:
This must keep OS-specific configs since it references the core assembly which is different per-os.